Misc/NEWS.d/next/Core and Builtins/2023-07-16-15-02-47.gh-issue-104090.oMjNa9.rst

```diff
@@ -0,0 +1,2 @@
+Updated Resource Tracker to return exit code based on resource leaked found
+or not
```

Comment:
```suggestion
The multiprocessing resource tracker now exits with status code 1 if a resource leak was detected.
It still exits with status code 0 otherwise.
```

Reply:
Done, I've updated the text